python/Ice/greeter/server_asyncio/main.py

```diff
@@ -34,11 +39,7 @@ def coroutineExecutor(coroutine):
         adapter.activate()
         print("Listening on port 4061...")
 
-        try:
-            # Wait until the event completes, which only occurs when it is canceled by Ctrl+C.
-            exit_event = asyncio.Event()
-            await exit_event.wait()
-        except asyncio.exceptions.CancelledError:
-            print("Caught Ctrl+C, exiting...")
+        # Wait until the communicator is shut down. Here, this occurs when the user presses Ctrl+C.
```

Comment:
We should add a comment about `Ice.wrap_future`, as it's not obvious. I assume it means we create an asyncio-compatible future from an Ice fture?

Reply:
Not clear why we have Ice.wrap_future, when there is asyncio.wrap_future for doing this.

https://docs.python.org/3/library/asyncio-future.html#asyncio.wrap_future